### ISSUE TITLE
Further instruction needed

### ISSUE THREAD

**Issue by libxx1** [raspberrypilearning/documenting-your-code#15]
URL: /en/projects/documenting-your-code/6

Project "Documenting your Code". 
Sphynx 3.4.2 Quickstart doesn't ask to activate Autodoc. It has to be manually activated in the config.py file by adding extensions = ['sphinx.ext.autodoc'].

**Comment by martinohanlon** [comment]
Yep, confirm. The sphinx-autostart utility has changed (pretty significantly). This step https://projects.raspberrypi.org/en/projects/documenting-your-code/6 needs to be updated to reflect the new sphinx-autostart utility (its much simpler now :) and as the feedback states you need to add the autodoc extension to the config.
